templates/test-source-image.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM src
+
+COPY oc /usr/bin/oc
+COPY --from=registry.svc.ci.openshift.org/openshift/knative-v__SERVING_VERSION__:knative-serving-src
```

Comment:
Would it make sense to use `${SERVING_VERSION}` instead of `__SERVING_VERSION__ ` this way it would be clear that it's going to be replaced. And same approach for other places in this PR.

Reply:
I didn't use `${SERVING_VERSION}` in Dockerfiles, because it's a legal Docker expression. If it slips in future someone could overlook it, theoretically. It may also be confusing.

We could use `#{SERVING_VERSION}` or stick with `__SERVING_VERSION__`.

What's your opinion?

Reply:
OK. I get it. Maybe it should be something that Docker doesn't parse. Either `#{}` or `{{ }}`. But it probably doesn't matter.

Reply:
I opt to stay with `__VARNAME__` style. I tried other styles, and they break syntax coloring for Dockerfiles.